development/mimir-monolithic-mode/config/datasource-mimir.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: 1
+datasources:
+- name: Mimir
```

Comment:
Should we make this datasource the default? (so set `isDefault: true`)

Reply:
Good idea - I'll also make the mimir datasource default in the microservices docker-compose, for consistency.